### PR TITLE
Fix deprecation warnings

### PR DESCRIPTION
**Pull request by ChrisdeWolf**
* ~resolve moderate severity npm audit warning with tough-cookie override~
* fix deprecation warning message for tests by updating Buffer
* add package-lock to .gitignore

### CODE REVIEW

File: package.json
Comment (twolfson) [inline package.json:31]: Is it wise to put an `overrides` in a package? This seems like people should do on a per-project basis, not me as a package maintainer

Comment (twolfson) [reply package.json:31]: i.e. Thoughts on making this as a workaround explained in #93 until `get-pixels` puts out a proper release?

Comment (ChrisdeWolf) [reply package.json]: Thats a fair workaround. Definitely your call as the package maintainer - removed the override.

Comment (twolfson) [reply package.json]: Cool, thanks! I'll post a comment to #93 with attribution to you then =)

**Review by twolfson — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
